instrumentation/spring/spring-boot-autoconfigure/build.gradle.kts

```diff
@@ -41,6 +41,10 @@ dependencies {
   compileOnly("io.opentelemetry:opentelemetry-exporter-otlp")
   compileOnly("io.opentelemetry:opentelemetry-exporter-zipkin")
   compileOnly(project(":instrumentation-annotations"))
+  compileOnly(project(":instrumentation:log4j:log4j-appender-2.17:library"))
+  compileOnly("org.apache.logging.log4j:log4j-core:2.17.0")
+  compileOnly(project(":instrumentation:logback:logback-appender-1.0:library"))
+  compileOnly("ch.qos.logback:logback-classic:1.0.0")
```

Comment:
what do you think about pulling in the appender instrumentation by default, similar to the other library instrumentations?

Reply:
created #8925 to track/discuss further, will merge this PR as is for the upcoming 1.28.0 release